packages/data-wizard/src/statistics/index.ts

```diff
@@ -260,3 +260,142 @@ export function coefficientOfVariance(array: number[]): number {
   const arrayMean = mean(array);
   return stdev / arrayMean;
 }
+
+/**
+ * Return the sum of the array by specific measure.
+ * @param array - The array to process
+ * @param measure - The selected measure
+ */
+export function sumBy(data: any[], measure: string) {
+  return data.map(typeof measure === 'function' ? measure : (val) => val[measure]).reduce((acc, val) => acc + val, 0);
+}
+
+/**
+ * Return the count of the array by specific measure.
+ * @param array - The array to process
+ * @param measure - The selected measure
+ */
+export function countBy(data: any[], measure: string) {
+  return data.filter((item) => measure in item).length;
+}
+
+/**
+ * Return the maximum of the array by specific measure.
+ * @param array - The array to process
+ * @param measure - The selected measure
+ */
+export function maxBy(data: any[], measure: string) {
+  return Math.max(...data.map(typeof measure === 'function' ? measure : (val) => val[measure]));
```

Comment:
Could `typeof measure === 'function'` become a function in `packages/data-wizard/src/utils/index.ts`?